pages/index.tsx

```diff
@@ -84,12 +84,13 @@ export default function Home({ events, webhooks }) {
             </p>
             <div className="mt-5 max-w-md mx-auto sm:flex sm:justify-center md:mt-8">
               <div className="rounded-md shadow">
-                <Link
-                  href="/create"
-                  onClick={onStartClick}
-                  className="w-full flex items-center justify-center px-8 py-3 border border-transparent text-base font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 md:py-4 md:text-lg md:px-10"
-                >
-                  Get started
+                <Link href="/create">
```

Comment:
Think we need to revert this change as well since the new Next.js doesn't allow nested `<a>` but then we are looking good!